tests/cell_test.cc

```diff
@@ -553,6 +553,12 @@ TEST_CASE("Cell is checked for 2D case", "[cell][2D]") {
       REQUIRE(strain_rate.size() == 3);
       for (unsigned i = 0; i < strain_rate.size(); ++i)
         REQUIRE(strain_rate(i) == Approx(0.).epsilon(Tolerance));
+
+      Eigen::VectorXd strain_rate_centroid =
```

Comment:
Can we do a test where we calculate strain rate? and have a non-zero value? Open this as a new issue. No need to fix in this version.